src/bootstrap.js

```diff
@@ -15,3 +15,4 @@ import 'bootstrap/js/transition';
 // Needed for a responsive navbar.
 import 'bootstrap/js/collapse';
 import 'bootstrap/js/dropdown';
+import 'bootstrap/js/button';
```

Comment:
I'm hoping that we can eventually do away with these Bootstrap plugins (getodk/central#1278), so I feel like we should be cautious before adding a new one.

- What exactly is the `button` plugin doing? How hard would it be to reimplement it ourselves in Vue?
- Is it an issue that we're styling radio inputs like buttons? Is that leading to extra complexity? I like the idea of radio inputs, but I'm open to replacing them with pure/simple buttons if they're forcing us to add a new Bootstrap plugin.